crates/burn-import/src/burn/node/max_pool1d.rs

```diff
@@ -0,0 +1,158 @@
+        let name = &self.field.name;
+        let kernel_size = self.config.kernel_size.to_tokens();
+        let strides = self.config.stride.to_tokens();
+        let padding = self.config.padding.to_tokens();
+        let dilation = self.config.dilation.to_tokens();
+        let tokens = quote! {
+            let #name = MaxPool1dConfig::new(#kernel_size)
+                .with_stride(#strides)
+                .with_padding(#padding)
+                .with_dilation(#dilation)
+                .init();
+        };
+
+        Some(tokens)
+    }
+
+    fn forward(&self, scope: &mut Scope, node_position: usize) -> TokenStream {
+        let input = scope.tensor_use_owned(&self.input, node_position);
+        let output = &self.output.name;
+        let field = &self.field.name;
+
+        quote! {
+            let #output = self.#field.forward(#input);
+        }
+    }
+
+    fn register_imports(&self, imports: &mut BurnImports) {
+        imports.register("burn::nn::PaddingConfig1d");
+        imports.register("burn::nn::pool::MaxPool1d");
+        imports.register("burn::nn::pool::MaxPool1dConfig");
+    }
+
+    fn into_node(self) -> Node<PS> {
+        Node::MaxPool1d(self)
+    }
+
+    fn field_serialize<S: serde::Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        S::serialize_none(serializer)
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::burn::{graph::BurnGraph, node::test::assert_tokens, TensorType};
+    use burn::{
+        nn::{pool::MaxPool1dConfig, PaddingConfig1d},
+        record::FullPrecisionSettings,
+    };
+
+    #[test]
+    fn test_codegen() {
+        let mut graph = BurnGraph::<FullPrecisionSettings>::default();
+
+        graph.register(MaxPool1dNode::new(
+            "max_pool1d",
+            TensorType::new_float("input", 3),
+            TensorType::new_float("output", 3),
+            MaxPool1dConfig::new(3)
+                .with_stride(1)
+                .with_padding(PaddingConfig1d::Valid)
+                .with_dilation(1),
+        ));
+
+        graph.register_input_output(vec!["input".to_string()], vec!["output".to_string()]);
+
+        let expected = quote! {
+            use burn::{
+                module::Module,
+                tensor::{backend::Backend, Tensor},
+            };
+            use burn::nn::PaddingConfig1d;
+            use burn::nn::pool::MaxPool1d;
+            use burn::nn::pool::MaxPool1dConfig;
+
+            #[derive(Module, Debug)]
+            pub struct Model <B: Backend> {
+                max_pool1d: MaxPool1d,
+                phantom: core::marker::PhantomData<B>,
+                device: burn::module::Ignored<B::Device>,
+            }
+
+            impl<B: Backend> Model <B> {
+                #[allow(unused_variables)]
+                pub fn new(device: &B::Device) -> Self {
+                    let max_pool1d = MaxPool1dConfig::new(3)
+                        .with_strides(1)
```

Comment:
`.with_strides(1)` should bd `.with_stride(1)` 

Currently the test is failing.